package.json

```diff
@@ -34,7 +34,7 @@
 		"eslint-plugin-jsx-a11y": "^6.6.1",
 		"eslint-plugin-react": "^7.31.10",
 		"eslint-webpack-plugin": "^3.2.0",
-		"stylelint": "^16.3.1",
+		"stylelint": "~16.3.1",
```

Comment:
16.4 at least also supports node 18: https://github.com/stylelint/stylelint/blob/16.4.0/package.json#L250

We shouldn't be limiting to 16.3 when later versions support the version of node we're using.